### PR TITLE
changed some typos

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2434

### CODE REVIEW

**Comment by mypil** [comment]
@kenwith - would you be able to merge the content of this PR? This is just for a typo issue based on issue #2434.

Thank you.

**Comment by kenwith** [comment]
On line 120 it looks like there is no space before the last sentence.

File: windows/deployment/update/waas-overview.md [hunk 1]
```diff
@@ -117,8 +117,7 @@ The concept of servicing channels is new, but organizations can use the same man
...
-Once the latest release went through pilot deployment and testing, you choose the timing at which it goes into broad deployment.
+In the Semi-Annual servicing channel, feature updates are available as soon as Microsoft releases them. Windows 10, version 1511, had few servicing tool options to delay feature updates, limiting the use of the Semi-Annual servicing channel. Windows 10, version 1607 and onward, includes more servicing tools that can delay feature updates for up to 365 days. This servicing model is ideal for pilot deployments and testing of Windows 10 feature updates and for users such as developers who need to work with the latest features immediately.Once the latest release has gone through pilot deployment and testing, you will be able to choose the timing at which it goes into broad deployment.
```
Comment (mypil) [inline windows/deployment/update/waas-overview.md]:
```suggestion
In the Semi-Annual servicing channel, feature updates are available as soon as Microsoft releases them. Windows 10, version 1511, had few servicing tool options to delay feature updates, limiting the use of the Semi-Annual servicing channel. Windows 10, version 1607 and onward, includes more servicing tools that can delay feature updates for up to 365 days. This servicing model is ideal for pilot deployments and testing of Windows 10 feature updates and for users such as developers who need to work with the latest features immediately. Once the latest release has gone through pilot deployment and testing, you will be able to choose the timing at which it goes into broad deployment.
```

[the remark this answers is not in the document]
Comment (kenwith) [reply windows/deployment/update/waas-overview.md]: @VLG17 - do you see a "Commit suggestion" button in this conversation thread? I am still getting familiar with this 'suggested change' feature and trying to see when the 'commit suggestion' shows up.

[the remark this answers is not in the document]
Comment (VLG17) [reply windows/deployment/update/waas-overview.md]: @kenwith yes i see a "Commit suggestion" drop down!

**Comment by mypil** [comment]
@kenwith - changes have been made to line 120. Kindly check and merge the content or let us know if there's something else that needs to be done. Thank you.

**Comment by kenwith** [comment]
Merged, thanks. Seems like the GitHub feature for 'suggest change' works pretty well. :)

**Pull request merged**